src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -61,21 +61,23 @@ def test_container_basics(self):
         self.container.get_params(self)
         self.container.create()
 
+	# TODO the cont info is out of date (see C version daos_cont_info_t)
```

Comment:
(pylint-fixme) TODO the cont info is out of date (see C version daos_cont_info_t)